server/database/seed.ts

```diff
@@ -35,10 +35,10 @@ Promise.all(movies.map((elem: movieObj, idx: number) => {
   // };
   // axios(options)
 
-  // Promise.all(movies.slice(0,5).map((elem: movieObj, idx: number) => {
-  //   return addMovie(elem, 1);
-  // }))
-  //   .then(() => console.log('all should be saved?'));
+  Promise.all(movies.slice(0,5).map((elem: movieObj, idx: number) => {
+    return addMovie(elem, 1);
+  }))
+    .then(() => console.log('all should be saved?'));
```

Comment:
I thought we weren't using this part any more?

Reply:
I put it back in for the moment when I was testing the profile, but since the movies part is going now, I think it will be safe to get rid of.